addons/mail/static/tests/discuss/call/ptt_ad_banner.test.js

```diff
@@ -32,6 +33,7 @@ test("display banner when ptt extension is not enabled", async () => {
     await click("button", { text: "Push to Talk" });
     await click("[title*='Close Chat Window']");
     await click("button", { text: "Start a meeting" });
+    await animationFrame();
```

Comment:
What's the point of this? The following `click` is supposed to wait for the element to be in DOM anyway. Adding an extra delay shouldn't make a difference.